recipe/meta.yaml

```
@@ -1,22 +1,25 @@
 {% set name = "arcticdb" %}
-{% set version = "5.1.2" %}
+{% set version = "5.2.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/man-group/ArcticDB/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: f088b8d15e5d1ccad53938bf639ac2848bfbdac29b83ced45707208ce61afbfe
-  patches:
-    - patches/0001-Skip-pyarrow.patch
-    - patches/0002-maint-fmt-11-support.patch
+  sha256: a9f18f87ec5d0f6e49945c204c85134c5eedfcfe92ff74126b317917015d19c0
 
 build:
-  # Only build for Linux and Python 3.9
-  skip: true  # [not linux]
-  skip: true  # [not py==39]
-  number: 4
+  # We skip the build on Windows because of linkage problems with libprotobuf.
+  # See: https://github.com/man-group/ArcticDB/pull/449
+  skip: true  # [win]
+  # Skip all builds but the ones for Linux for the `libevent-2.1.10` specification.
+  # See: https://github.com/conda-forge/arcticdb-feedstock/blob/cfb0ce12a21703d55389d1147a689790447b0c03/recipe/meta.yaml#L15-L17
+  skip: true  # [libevent == "2.1.10" and osx]
+  # TODO: the specific on python does not work for now, we need to investigate why.
+  # skip: true  # [libevent == "2.1.10" and py>39]
+  # skip: true  # [libevent == "2.1.10" and py<39]
```

A suggestion: I think this is simpler.
```suggestion
  # Only build for Linux and Python 3.9
  skip: true  # [not linux]
  skip: true  # [not py==39]
```